src/site/markdown/index.md

```diff
@@ -497,57 +497,6 @@ in `MessageDecoder`.
     }
 
 
-
-### JSON-B Extra
```

Comment:
maybe keep it and explain:

1. very high level what is in the spec
2. how to migrate from extras module to the standard impl

Reply:
don't know if migrating from jsonb-extras to jsonb3 will even be something users want to do - generated JSON format is different (`{"_type": "bla", "_value": {"someValue": 1}}` vs `{"_type": "bla", "someValue": 1}`)

Don't really want to add support for this format in jsonb module directly. It's out of spec and jsonb-extras was portable and is probably used on top of other jsonb impls
Maybe jsonb-extras shouldn't actually be deleted? It was pretty leightweight anyways

Reply:
:), fine to revert the deletion, while the feature is there all good for me.

Reply:
reverted the deletion + updated docs that extras polymorphism _shouldn't_ be used anymore